tests/conftest.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Autobot.
+# Copyright (C) 2015-2019 CERN.
+#
+# Autobot is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test configuration file."""
+
+import pytest
+import yaml
+
+
+@pytest.fixture(scope="session")
+def mock_info(tmpdir_factory):
+    """Mock info for configuration testing."""
+    mock = {
```

Comment:
I think here we need a much smaller config, which we're going to use for tests. A list with fake explicit names for each role would also help later on, e.g. `maintainer-foo` responsible for repositories `foo-1`, `foo-2`; `maintainer-bar` responsible for `bar1`, `bar2`; `maintainer-mix` responsible for `foo1`, `bar2`, `baz1`.